libnavui-voice/api/current.txt

```diff
@@ -2,8 +2,12 @@
 package com.mapbox.navigation.ui.voice.api {
 
   public interface AudioFocusDelegate {
-    method public boolean abandonFocus();
-    method public boolean requestFocus();
```

Comment:
This is a breaking change, which is not allowed at current stage of development.

Reply:
We might need to consider a completely new interface and a different `MapboxVoiceInstructionsPlayer` constructor that accepts it to provide backwards compatibility.

Reply:
Not sure that we need care about it. This is currently being used to override the default behavior (which is implemented in the constructor). This is primarily needed for developers who want to control focus delay behavior.

Reply:
If there are developers that provide a custom behavior and implement this interface, this change would break their projects and wouldn't be backwards compatible. We'll need to find a backwards compatible way of providing the new callback.

Reply:
Where are we here @VitaminPSG?

Reply:
I have no idea how to provide backwards compatibility